packages/ember-views/lib/views/view.js

```diff
@@ -1022,6 +1022,8 @@ var View = CoreView.extend({
     @return Ember.View
   */
   nearestChildOf: function(klass) {
+    Ember.deprecate("nearestChildOf has been deprecated.\nFor existing behavior, use nearestOfType.");
```

Comment:
Eh. The existing behavior doesn't work. At all. So I might just change this to: `"nearestChildOf has been deprecated"`